docs/changelog.md

```
@@ -0,0 +1,35 @@
+
+### Changed
+
+### Removed
+- **Web UI:** Remove unused settings page
+
+### Fixed
+-->
+
+## v0.2.0 -- Initial public release
+
+### Added
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)